### PR TITLE
OOP-School-Library set up associations

### PR DESCRIPTION
**Pull request by 1GORDON**
# In this project, I achieved the following
- Created a class Classroom with @label instance variable, should be initialized in the constructor.
- Updated Class Student.
- Created a class Book with the @title and @author instance variables, which should be initialized in the constructor.
- Created a class Rental with the @date instance variable, which should be initialized in the constructor.
- Updated Person Class.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_students(student)
+    @students.push(student)
+    student.classroom = self
+  end
```
Comment (Dipeshtwis) [inline classroom.rb]: [ OPTIONAL]

- As per ruby convention, keep the method name **singular**

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```
Comment (Dipeshtwis) [inline rental.rb]: - [x] Getters and Setters for `book` and `person` instance variables should be present. As we are going to `get the value` as well as we need to `set the value`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @1GORDON, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classroom.rb (hunk 1 above)
Comment (1GORDON) [reply classroom.rb]: Fixed

File: rental.rb (hunk 2 above)
Comment (1GORDON) [reply rental.rb]: Fixed

**Review by addod19 — APPROVED** [review]
### Hi @1GORDON, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**